yamoveset/panels/main.py

```diff
@@ -368,3 +375,69 @@ def on_paste(self, _):
         # with MultiMessageDialog(
         #         self, 'The following entries in the listed files have been changed:', msg, changed_msg, wx.OK) as dlg:
         #     dlg.ShowModal()
+
+    def on_add(self, _):
+
+
+        if not self.parent.copied:
+            return
+
+        copied = pickle.loads(self.parent.copied)
+
+
+
+        selected_data = [item for item in copied]
+
+
+
+        # Paste entries
+        selected_values = []
+        copied_values = []
+        changed_values = defaultdict(list)
+
+        for n, copied_data in enumerate(copied):
+            selected_values.append((selected_data[n].index, selected_data[n].get_static_values()))
+            copied_values.append(copied_data.get_static_values())
+
+        for selected_val, copied_val in zip(selected_values, copied_values):
+            # Example:
+            # Item type: Animation
+            # entry: Index
+            # dependency: Type
+            # depend_value: 5
+            # entry_values: {1, 2, 3}
+            for item_type, v1 in copied_val.items():
+                if item_type not in [Animation, Hitbox, Camera]:
+                    continue
+                for entry_pair, v2 in v1.items():
+                    for depend_value, entry_values in v2.items():
+                        # Skip if dependency isn't a character
+                        if item_type.dependencies[entry_pair][depend_value] != 'Character':
+                            continue
+                        if not self.get_changed_values(
+                                changed_values, item_type, entry_pair, depend_value, entry_values,
+                                selected_val, selected_data):
+                            return
+
+
+        # Add BAC Entry
+        for n, copied_data in enumerate(copied):
+            index = len(self.bac.entries)
+            new_entry = Entry(self.bac, index)
+            root = self.entry_list.GetRootItem()
+            new_entry.paste(copied_data, self.links)
+            self.bac.entries.insert(index, new_entry)
+            self.entry_list.AppendItem(
+                root, f'{new_entry.index}: {KNOWN_ENTRIES.get(new_entry.index, "Unknown")}', data=new_entry)

```

Comment:
are we inserting or are we appending?

Reply:
my bad, should be append
leftover from a test